src/routes/(app)/strafbakken/+server.ts

```diff
@@ -0,0 +1,40 @@
+import type { RequestHandler } from "./$types";
+import db from "$lib/server/db";
+import { json } from "@sveltejs/kit";
+
+export const DELETE: RequestHandler = async ({ request }) => {
+  const { user }: { user: number } = await request.json();
+
+  const strafbak = await db.strafbak.findFirst({
+    where: {
+      receiverId: user,
+      dateDeleted: null,
+    },
+    orderBy: {
+      dateCreated: "desc",
+    },
+  });
+
+  if (!strafbak)
+    return json(
+      {
+        message: `User ${user} heeft geen strafbakken`,
```

Comment:
doe je iets met deze json?

Reply:
op het moment niet, maar is wel nice voor de integratie met discord uiteindelijk